openapi_python_client/__init__.py

```diff
@@ -78,9 +77,9 @@ def __init__(
             keep_trailing_newline=True,
         )
 
-        project_name_base: str = config.project_name_override or f"{utils.kebab_case(openapi.title).lower()}"
+        project_name_base: str = config.project_name_override or f"{utils.kebab_case(openapi.info.title).lower()}"
         self.project_name = project_name_base + config.project_name_suffix
-        self.package_name: str = config.package_name_override or self.source_name
```

Comment:
I think I changed that to `source_name` to have nicer cli. did you revert or I forgot something?

Reply:
It's still the same I think. The `package_name_override` is set to source name in cli: https://github.com/dlt-hub/dlt-init-openapi/blob/new-parser-experiment/openapi_python_client/cli.py#L90